### DISCUSSION TITLE
Nitro на продакшене

### DISCUSSION THREAD

**Discussion by surguh** [devind-team/dcis#702] [General]
При сборке клиента ситуация аналогичная #687 . Исследуя варианты решил попробовать собрать релиз на nitro сервере и он заработал. Может перейти на него?
Плюсы перехода:
Собранный сервер не требует дополнительных зависимостей (node_modules)
Весит ~20Мб. (На nuxt2 сервере больше 1Gb с dev зависимостями)
Минусы:
Не известно на сколько он стабилен (на win11 все работает нормально на первый взгляд)

**Comment by Luferov** [comment]
С нитро идея отличная, но в продакшене есть проблема, он не парсит динамические пути
если у нас есть путь
site.ru/auth/cbias/:uid
то если перейти по этой ссылке, то сервер завалиться. Попробуйте проверить, если это не так и проблема устранена, то это отлично. Возможно, для SPA работает на ура

**Reply by Luferov** [reply]
Собрал с нитро
![image](https://user-images.githubusercontent.com/26242554/187170529-b6bd4651-68df-49c4-bda7-87026027dd17.png)

**Reply by surguh** [reply]
> Собрал с нитро ![image](https://user-images.githubusercontent.com/26242554/187170529-b6bd4651-68df-49c4-bda7-87026027dd17.png)

Вспомнил, я один раз такое поймал. Надо куки почистить. А так на ссылку http://localhost:3000/dcis/projects/UHJvamVjdFR5cGU6NQ== переходит нормально

**Reply by Luferov** [reply]
Может быть они все починили, тогда это очень и очень хорошо! Давайте попробуем, если что обратно hotfix откатим
